Assets/MixedRealityToolkit/Extensions/CameraExtensions.cs

```diff
@@ -4,7 +4,7 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit
+namespace Microsoft.MixedReality.Toolkit.Tests.Extensions
```

Comment:
This doesn't seem like a script meant for the `Tests` namespace

Reply:
good catch, I had the file named the same